src/main/java/com/faforever/client/player/PlayerService.java

```diff
@@ -60,6 +60,7 @@
   private final FafService fafService;
   private final UserService userService;
   private final EventBus eventBus;
+  private final HashMap<Integer, List<Player>> playersByGame = new HashMap<>();
```

Comment:
Don't initialize field here, by convention we always do in the constructor.

Reply:
Ok just curious. What convention is that?